test/kino_db/sql_cell_test.exs

```diff
@@ -59,8 +50,45 @@ defmodule KinoDB.SQLCellTest do
       "connections" => [^connection],
       "connection" => ^connection
     })
+  end
 
-    send(conn_pid, :stop)
+  test "keeps the currently selected connection if not available in binding" do
+    attrs = %{"connection" => %{"variable" => "conn1", "type" => "postgres"}}
+    {kino, _source} = start_smart_cell!(SQLCell, attrs)
+
+    conn_pid = spawn_fake_postgrex_connection()
+
+    binding = [conn: conn_pid]
+    # TODO: Use Code.env_for_eval on Elixir v1.14+
+    env = :elixir.env_for_eval([])
+    SQLCell.scan_binding(kino.pid, binding, env)
+
+    current_connection = %{variable: "conn1", type: "postgres"}
+    connection = %{variable: "conn", type: "postgres"}
+
+    assert_broadcast_event(kino, "connections", %{
+      "connections" => [^connection],
+      "connection" => ^current_connection
+    })
+  end
+
+  test "updates the selected connection type when the variable changes" do
+    attrs = %{"connection" => %{"variable" => "conn", "type" => "sqlite"}}
+    {kino, _source} = start_smart_cell!(SQLCell, attrs)
+
+    conn_pid = spawn_fake_postgrex_connection()
+
+    binding = [conn: conn_pid]
+    # TODO: Use Code.env_for_eval on Elixir v1.14+
+    env = :elixir.env_for_eval([])
```

Comment:
Kino requires Elixir v1.14, so we can bump it here too. I will push directly to main after the PR.